### DISCUSSION TITLE
Why aren't my pull requests merged?

### DISCUSSION THREAD

**Discussion by psprint** [zdharma-continuum/zinit#567] [General]
Hi,
I'm trying to develop zinit from a "back seat", as I've lost control over the project. That's fine, IMO the community control over the project is great. That said, I'm having problems with getting my PRs merged… I'm waiting over a **year** for some of them:

- #333,
- #406,
- #458 

Could the PRs be merged quicker? I've refreshed them so that there are no merge conflicts:
- #551,
- #550, 
- #548,
- #547,
- #468.

I don't know what else to do, as constant pinging the devs doesn't help…

**Comment by psprint** [comment]
@vladdoster @alichtman @pschmitt: ping

**Reply by alichtman** [reply]
I've switched over to using zcomet as a plugin manager and see no performance downgrade and no missing features for my setup. I will likely be resigning from the zinit project. I never intended on becoming an active maintainer, and don't have the time or desire to do so. If @pschmitt and @vladdoster do something similar (I haven't spoken to them about it yet), you could probably resume control of the project, as there are now other options that are just as fast and easier to configure, if zinit implodes for a third time.

Best,
Aaron

**Comment by psprint** [comment]
> you could probably resume control of the project

Oh yes, I would very happily do!

**Reply by psprint** [reply]
i'm trying my luck with z-shell/zi, https://github.com/orgs/z-shell/discussions/280

**Comment by psprint** [comment]
@vladdoster @alichtman @pschmitt: Or, not, I'll be setting up my own zinit fork tomorrow. @z-shell can be some China or other such project.
